spec/compiler/parser/to_s_spec.cr

```diff
@@ -108,4 +108,9 @@ describe "ASTNode#to_s" do
   expect_to_s %(1 <= 2 <= 3)
   expect_to_s %((1 <= 2) <= 3)
   expect_to_s %(1 <= (2 <= 3))
+
+  expect_to_s %(case\nwhen 1\nwhen 2\nend)
```

Comment:
I'm wondering, what is the use of a `case` not followed by an expression, shouldn't this be disallowed?